aytests/sles12.list

```diff
@@ -17,3 +17,5 @@ ifcfg_networking.sh            # configures network interfaces (bsc#949193)
 restart_services.sh            # do no restart dbus and wickedd* services (bsc#944349)
 init_script.sh		       # download and execute init scripts (bnc#961320)
 check_schema.sh                # check generated AutoYaST profile (bsc#954412)
+report_unsupported_sections.sh # report unsupported sections
```

Comment:
I found this PR: https://github.com/yast/yast-autoinstallation/pull/117 which references to bsc#925381. I'm adding that reference.